app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100, hidden=False):
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self):
+        return f"{{Name: {self.name}," \
+               f" Health: {self.health}," \
+               f" Hidden: {self.hidden}}}"
+
+
+class Herbivore(Animal):
+    def hide(self):
+        if self.hidden is False:
+            self.hidden = True
+        else:
+            self.hidden = False
```

Comment:
You can change the `bool` value in one line:

Good example:

```python
is_married = True
is_married = not is_married
```

Reply:
i cannot chenge, I already tried initially, but with this solution it gives an error. (AssertionError: Method 'hide' should change animal attribute 'hidden')
class Herbivore(Animal):
    def hide(self):
        if self.hidden is False:
            self.hidden = not self.hidden

Reply:
![image](https://user-images.githubusercontent.com/110623134/189361175-7f0242a6-666a-478f-b1eb-834ac5fc9608.png)

Reply:
understood, I changed